GeoNames was missing an S

Moutony opened the pull request.

tommasoberlose commented:
@Moutony I think that Widget Align fits better because that option doesn't only change the text's alignment but also changes the layout of the widget. Right now it just moves the weather below but I was thinking about improving the widget when on the side.

Moutony commented:
I fixed like you said. Btw i changed align into alignment because it wasn't grammatically correct.
Align is a transitive verb so the correct word order would rather be: Align widget. But the word alignment is more oftenly used on web so i fixed it.

tommasoberlose commented:
Oh you're right! Thanks
